lib/util/clone.ts

```diff
@@ -1,5 +1,10 @@
 import { quickStringify } from './stringify';
 
+/**
+ * Creates a deep clone of an object.
+ * @deprecated Use {@link structuredClone} instead.
+ * @param input The object to clone.
+ */
```

Comment:
Is it not possible to fully remove?

Reply:
Not right now. I need to figure out how to get `structuredClone` to work with `nock` and `dequal`.

I'd prefer to merge this now, and open a PR later to fix those issues.